code/style.css

```diff
@@ -1,5 +1,156 @@
 /* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
 
+img {
+  font-size: 14px;
+}
 body {
-  background-color: lightblue;
+  font-family: "Helvetica Neue", Arial, Helvetica, sans-serif;
+  margin: 100px 0px;
+  background-color: rgb(252, 253, 254);
+}
+h1 {
+  display: left;
+  padding: 10px;  
+  margin-top: -100px;
+  margin-left: -200px;
+  float:left;   
+}
+h2 {
+  display: left;
+  padding: 10px;  
+  margin-top: -40px;
+  margin-left: -200px;
+  float:left;   
+}
+header {
+  background-image: url('pic1.jpg');
```

Comment:
Super nice with a background image 🤩

Reply:
If you give the header a smaller `padding-top` you don't have to use negative values for the `margin` of the headings :)